compiler/commands.nim

```diff
@@ -670,7 +669,6 @@ proc processSwitch*(switch, arg: string, pass: TCmdLinePass, info: TLineInfo;
     processOnOffSwitchG(conf, {optThreads}, arg, pass, info)
     #if optThreads in conf.globalOptions: conf.setNote(warnGcUnsafe)
   of "tlsemulation": processOnOffSwitchG(conf, {optTlsEmulation}, arg, pass, info)
```

Comment:
```suggestion
  of "tlsemulation": processOnOffSwitchG(conf, {optTlsEmulation}, arg, pass, info)
  of "taintmode": discard
```

so that code that used `--taintmode` etc still works, but with a warning

the warning should be introduced in future work (refs https://github.com/nim-lang/Nim/pull/15423/files#r558480731)

Reply:
Can you add a TODO to the code as a comment?